stream_alert/classifier/clients/firehose.py

```diff
@@ -0,0 +1,410 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from collections import defaultdict
+import io
+import json
+import re
+
+import backoff
+import boto3
+from botocore.exceptions import ClientError
+from botocore.vendored.requests.exceptions import ConnectionError, Timeout
+import jsonlines
+
+from stream_alert.shared import CLASSIFIER_FUNCTION_NAME as FUNCTION_NAME
+from stream_alert.shared.helpers import boto
```

Comment:
Can we rename `boto` to `boto_config`? It is just client config for boto client.

Reply:
I named it that so it could theoretically be expanded to other things.. for instance I was also thinking about adding factory function that returns a client/resource for a given service (and uses this default config). is it okay to leave as is?

Reply:
sgtm!

Reply:
addressed the confusion via updating the import/usage of this helper :)